### PR TITLE
[CLI] Add Login subcommand

### PR DESCRIPTION
**Pull request by IvanPombo**
TLDR: Add a subcommand to CLI to allow for login/sign-in into Inductiva API.

## Description

With the purpose of facilitating getting an API key, this PR introduces a new subcommand to the CLI that allows to login into the console to register yourself in the platform, get an API Key and allow to regenerate it.

## Implications

The name attributed to the subcommand is `login` because it allows for much more functionality than just register ourselves in the platform. For example, we can use `inductiva login` to login into the console and fetch the API key, or regenerate it.

Still not completely sure, this is the correct name, but I didn't like `register` either due to the above implications.

### CODE REVIEW

File: inductiva/_cli/cmd_login/__init__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        cmd = ["cmd", "/c", "start", url.replace("&", "^&")]
+    else:
+        raise RuntimeError(f"Unsupported system: {system}")
+    subprocess.run(cmd, check=True)
+
+
+def register(root_parser):
+    parser = root_parser.add_parser(
+        "login",
+        help="Login to get an Inductiva API Key.",
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_login/__init__.py:30]:
```suggestion
        help="Register to create an Inductiva API key.",
```

**Comment by Pbarbosa92** [comment]
**QA**

✅ Ran inductiva login and was correctly redirected to the console website.

**Review by AugustoPeres — APPROVED** [review]

File: inductiva/_cli/cmd_login/__init__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    parser.description = (
+        "Login into the Inductiva console to register in the platform and "
+        "obtain\nyour API key. Currently, this method requires an Google "
+        "email.\nFurther, you can login to generate a new API key when needed."
+        "\n\nTo use the API set the key as an environment variable:\n "
+        "\texport INDUCTIVA_API_KEY=<your_api_key>")
```
[the remark this answers is not in the document]
Comment (sergiomsantos) [reply inductiva/_cli/cmd_login/__init__.py:38]: if so, the description requires an overview (login to register to the login to get the key)
```suggestion
        "Login into the Inductiva console to register in the platform and "
        "obtain\nyour API key. Currently, this method requires a Google "
        "email.\nFurther, you can login to generate a new API key when needed."
        "\n\nTo use the API set the key as an environment variable:\n "
        "\texport INDUCTIVA_API_KEY=<your_api_key>")
```
